config/config.exs

```diff
@@ -1,4 +0,0 @@
-use Mix.Config
```

Comment:
Is it a good idea to remove this file?
For this project to run it needs this and the values are just placeholders. Since this file is in the git-ignore no sensitive info should be committed anyways.

Reply:
I'd rather not remove it anyway.
Maybe add a config.exs.example and add something to the README explaining to rename it and add the tokens.